example/glue/ConfigurationDelegate.hpp

```diff
@@ -175,6 +175,19 @@ class MM_ConfigurationDelegate
 	 */
 	MM_GCPolicy getGCPolicy() { return _gcPolicy; }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * This method is called to adjust the checkpointgcThreadCount if not
+	 * specified by the user, or verify that the checkpointgcThreadCount is
+	 * not larger than the number of gc threads.
+	 * 
+	 * If the checkpointgcThreadCount is specified by the user and larger
+	 * than the number of gc threads, JVM throws a warning and continue.
```

Comment:
OMR code should not refer to JVM (Java VM), but just a VM. And the whole comment could be a little more relaxed/generic (just _suggesting_ what the method could be used for, in any possible language). As such, it sounds a bit too specific, targeting how Java VM exactly uses it.